examples/ftd_api_scripts/timerange-access-rule-association.py

```diff
@@ -0,0 +1,154 @@
+'''
+Copyright (c) 2019 Cisco and/or its affiliates.
+This software is licensed to you under the terms of the Cisco Sample
+Code License, Version 1.1 (the "License"). A copy of the License
+can be found in the LICENSE.TXT file of this software or at
+https://developer.cisco.com/site/license/cisco-sample-code-license/
+All use of the material herein must be in accordance with the terms of
+the License. All rights not expressly granted by the License are
+reserved. Unless required by applicable law or agreed to separately in
+writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied.
+'''
+
+import sys
+from ftd_api_resources.time_range_access_rule_association_utils import get_all_access_rules, \
+    post_access_rule, update_access_rule, get_access_rule
+from ftd_api_resources.timerange_utils import post_time_range
+from ftd_api_resources.access_token import get_access_token
+
+
+def timerange_access_rule_association_actions(host, port, user, passwd, parent_policy_id):
+    """
+    End to end example of code that updates an access rule.
+    Requires Python v3.0 or greater and the reqeusts library.
+    You must update the values for host, port, user, and password to connect to your device.
```

Comment:
Replace this line with params list

Reply:
Done, addressed in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/7